### PR TITLE
chore(package): add webpack `peerDependency`

### LINKED ISSUE #168: 1.0.0-beta.0 does not build
I can't give much info (it's late here in France), but the last version does not work with webpack 1.15 on multiple machines with different node versions.
It fails on importing both `png` and `scss` files.

I have multiple messages similar to `Module not found: Error: Cannot resolve module 'file' in ...`

This is fixed by rolling back to version `0.11.2`.

The webpack config:
```js
var path = require('path');
var webpack = require('webpack');
var ExtractTextPlugin = require('extract-text-webpack-plugin');
var CopyWebpackPlugin = require('copy-webpack-plugin');

module.exports = {
    entry: [
        'XXXX.jsx',
        'XXXX.scss',
        'webpack/hot/dev-server',
        'webpack-dev-server/client?http://localhost:4000',
    ],
    output: {
        path: __dirname,
        filename: 'build/app.js',
    },
    resolve: {
        extensions: ['', '.js', '.jsx'],
        alias: {
            pdfmake: path.resolve(
                __dirname,
                './node_modules/pdfmake/build/pdfmake.js'
            ),
        },
    },
    module: {
        loaders: [
            {
                test: /\.jsx?$/,
                exclude: /node_modules/,
                loader: 'babel',
            },
            {
                test: /\.scss$/,
                loader: ExtractTextPlugin.extract('css!sass'),
            },
            {
                test: /\.(woff|woff2|eot|ttf|svg|otf)$/,
                loader: 'file?name=/build/fonts/[sha512:hash:base64:7].[ext]',
            },
            {
                test: /\.ya?ml$/,
                include: __dirname + '/build',
                loader: 'json-loader!yaml-include-loader',
            },
            {
                test: /\.json$/,
                loader: 'json-loader',
            },
            {
                test: /\.(jpe?g|png|gif|svg)$/i,
                loaders: [
                    'file?hash=sha512&digest=hex&name=/build/images/[hash].[ext]',
                ],
            },
        ],
    },
    plugins: [
        // Always expose NODE_ENV to webpack, in order to use `process.env.NODE_ENV`
        // inside your code for any environment checks; UglifyJS will automatically
        // drop any unreachable code.
        new webpack.DefinePlugin({
            'process.env': {
                NODE_ENV: JSON.stringify(process.env.NODE_ENV),
            },
        }),
        new ExtractTextPlugin('build/style.css'),
        new CopyWebpackPlugin([
            {
                from: 'src/assets/images/favicons/favicon.ico',
                to: 'build/favicon.ico',
            },
            {
                from: 'src/assets/images/favicons/favicon.png',
                to: 'build/favicon.png',
            },
            {
                from: 'src/assets/images/favicons/favicon-32.png',
                to: 'build/favicon-32.png',
            },
            {
                from: 'src/assets/images/favicons/favicon-64.png',
                to: 'build/favicon-64.png',
            },
            {
                from: 'src/assets/images/favicons/favicon-96.png',
                to: 'build/favicon-96.png',
            },
        ]),
        new webpack.HotModuleReplacementPlugin(),
    ],
    watchOptions: {
        poll: true,
    },
};

```

### PR DESCRIPTION
**Pull request by michael-ciniawsky**
Fixes #168

### CODE REVIEW

**Comment by michael-ciniawsky** [comment]
Sry, sloppy from my side I missed `test !== travis` :)

File: package.json [hunk 1]
```diff
@@ -27,20 +27,22 @@
...
   },
+  "peerDependencies": {
+    "webpack": "^2.0.0 || >= 3.0.0-rc.0 || ^3.0.0"
+  },
   "scripts": {
     "start": "npm run build -- -w",
     "build": "cross-env NODE_ENV=production babel src -d dist --ignore 'src/**/*.test.js'",
     "clean": "del-cli dist",
-    "clean:dist": "del-cli dist",
     "lint": "eslint --cache src test",
```
Comment (evilebottnawi) [inline package.json]: Hm, where `clean:dist` task?

Comment (michael-ciniawsky) [inline package.json]: `clean:dist` => `clean`

File: package.json [hunk 2]
```diff
@@ -27,20 +27,22 @@
...
     "lint-staged": "lint-staged",
     "prepublish": "npm run build",
     "prebuild": "npm run clean",
     "release": "standard-version",
     "security": "nsp check",
     "test": "jest",
-    "test:coverage": "jest --collectCoverageFrom='src/**/*.js' --coverage",
     "test:watch": "jest --watch",
+    "test:coverage": "jest --collectCoverageFrom='src/**/*.js' --coverage",
```
Comment (evilebottnawi) [inline package.json]: It is normal change position?

**Pull request merged**